stencils/pace/stencils/testing/translate.py

```diff
@@ -235,7 +235,19 @@ class TranslateGrid:
     fpy_model_index_offset = 2
     fpy_index_offset = -1
     composite_grid_vars = ["sin_sg", "cos_sg"]
-    edge_var_axis = {"edge_w": 1, "edge_e": 1, "edge_s": 0, "edge_n": 0}
+    vvars = ["vlon", "vlat"]
```

Comment:
[for this file]: Do you know if the jenkins test triggered from this PR runs c12_54_ranks_baroclinic_dycore_microphysics using all the logic here? Since you didn't mention verifying 54 ranks driver test, just wanted to make sure that we at least test this infrastructure on something more than 6 ranks.

Reply:
it works, yay!